api/google-search/src/main/java/com/google/search/googlesearch/controller/v1/api/SearchController.java

```diff
@@ -31,4 +44,14 @@
 
         return response;
     }
+
+    @PostMapping(produces = MediaType.APPLICATION_JSON_VALUE, value = "v1/addSearch")
+    public Search addSearchResult(@RequestBody Search search) {
+        return searchRepository.save(search); 
+    }
+
+    @GetMapping(produces = MediaType.APPLICATION_JSON_VALUE, value = "v1/getSearch")

```

Comment:
@jpukg 
is value fine `v1/getSearch` ????

Reply:
@karthikeyanpa12 @ramsunvtech  

Let's have the URL as `v1/search` for CRUD operations.

- For creating it's POST -> `v1/search`
- For reading it's GET -> `v1/search`
- For updating it's PUT -> `v1/search`
- For Deleted it's DELETE -> `v1/search`